app/bundles/CoreBundle/Views/Helper/page_actions.html.php

```diff
@@ -76,6 +78,12 @@
                 echo '  <i class="fa fa-'.$icon.'"></i> ' . $view['translator']->trans('mautic.core.form.' . $action) . "\n";
                 echo "</a>\n";
                 break;
+            case 'close':
+                $icon = 'remove';
+                echo '<a class="'.$btnClass.'" href="' . $view['router']->generate('mautic_' . $routeBase. '_action' ,array("objectAction" => "")) . '" data-toggle="' . $editMode . '"' . $editAttr . $menuLink . ">\n";
```

Comment:
In dev mode, this is throwing an exception because the router expects `objectAction` to have non-empty value. You can enter dev mode by using `localhost/index_dev.php/` rather than just `localhost` or `localhost/index.php` (or whatever the local domain is)